api/.env

```diff
@@ -36,5 +39,8 @@ MERCURE_PUBLISH_URL=http://mercure/hub
 MERCURE_JWT_SECRET=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjcmliZSI6WyJmb28iLCJiYXIiXSwicHVibGlzaCI6WyJmb28iXX19.LRLvirgONK13JgacQ_VbcjySbVhkSmHy3IznH3tA9PM
 ###< symfony/mercure-bundle ###
 
-MERCURE_SUBSCRIBE_URL=https://localhost:1338/hub
-VARNISH_URL=http://cache-proxy
+###> lexik/jwt-authentication-bundle ###
+JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
```

Comment:
Can you add a command in the Dockerfile, to generate those keys?

Reply:
The keys should not be in the Docker image, as it's data and/or secret, depending on how you look at it.

Reply:
Right for production, but should be autogenerated in dev. Maybe in the entrypoint?

Reply:
Yeah, it means adding openssl dependency to the php image, but that's fine by me.